ccd-definition/CaseEventToFields/DefendantBankAccounts.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
+    "CaseFieldID": "minusBeforeOverdrawnAmount",
+    "PageID": "DefendantBankAccounts",
+    "PageDisplayOrder": 11,
+    "PageFieldDisplayOrder": 1,
```

Comment:
These appear exactly similar to next file contents - ccd-definition/CaseEventToFields/response/partaccept/DefendantBankAccounts.json